packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -146,6 +146,7 @@ function AuthOptions({
   const [handleLoginCheck, setHandleLoginCheck] = useState<boolean>(null);
   const [chosenProvider, setChosenProvider] = useState<string>(null);
   const [isRegistration, setIsRegistration] = useState(false);
+  const [isLogin, setIsLogin] = useState(false);
```

Comment:
Isn't this exactly what isLoginFlow should do?
Feels like we now have 2 states for the same

Reply:
Yeah 😅
`isLoginFlow` is controlled via the `loginState` in `AuthContext`, so I would have to import it to update that state
https://github.com/dailydotdev/apps/blob/4585cc58ef9fb3c936139394ef9ba9486008dc60/packages/shared/src/components/auth/AuthModal.tsx#L72C22-L72C34

Reply:
Hmm if it's a mayhem let's just do this and have tech debt for later